packages/apps/kadena-docs/src/components/BottomPageSection/components/EditPage.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button } from '@kadena/react-components';
+
+import { FC } from 'react';
+
+interface IProps {
+  filename?: string;
+}
+
+const editRoot = process.env.NEXT_PUBLIC_GIT_EDIT_ROOT;
+
+export const EditPage: FC<IProps> = ({ filename }) => {
+  if (editRoot === null || filename === null) return null;
+  return (
+    <a href={`${editRoot}${filename}`} target="_blank" rel="noreferrer">
+      <Button title="Edit this page">Edit this page</Button>
+    </a>
```

Comment:
I'm a bit torn about the Button component. Do you think this is something the button component should be able to handle so you don't have to wrap it in an anchor or is it specific enough to not be handled?

Reply:
good point. had to make some changes to the button component, though.